bool kanyarodas
bool jobbra_kanyarodas
float32 fordulokor_sugar
float32[] pontok_x
float32[] pontok_y
float32[] objektum_kozeppont_x
float32[] objektum_kozeppont_y
float32 linear_vel
float32 angular_vel
bool acc_on_off
bool lock_on
float32 veh_vel_measured
bool object_detected
float32 avg_rel_vel_of_closest_interesting_object
float32 closest_interesting_object_x_range
float32 veh_vel_control_lin
float32 veh_vel_control_ang
float32 object_turning_radius
float32 closest_object_range
float32 object_speed
float32 control_sign
